Update `.quickstart.yml` and `README` (#11)
fivetran/dbt_aws_cloud_cost: README.md

    <a alt="PRs">
        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
    <a alt="Fivetran Quickstart Compatible"
        href="https://fivetran.com/docs/transformations/data-models/quickstart-management#quickstartmanagement">
        <img src="https://img.shields.io/badge/Fivetran_Quickstart_Compatible%3F-yes-green.svg" /></a>
</p>

This dbt package transforms data from Fivetran's Aws Cloud Cost connector into analytics-ready tables.

## Resources

- Number of materialized models¹: 5
- Connector documentation

>   - [Aws Cloud Cost connector documentation](https://fivetran.com/docs/connectors/applications/aws-cloud-cost)

Reviewer: ```suggestion
  - [AWS Cloud Cost connector documentation](https://fivetran.com/docs/connectors/applications/aws-cost-report)
```

>   - [Aws Cloud Cost ERD](https://fivetran.com/docs/connectors/applications/aws-cloud-cost#schemainformation)

Reviewer: ```suggestion
  - [AWS Cost & Usage Report](https://fivetran.com/docs/connectors/applications/aws-cost-report#syncoverview)
```

- dbt package documentation
  - [GitHub repository](https://github.com/fivetran/dbt_aws_cloud_cost)
  - [dbt Docs](https://fivetran.github.io/dbt_aws_cloud_cost/#!/overview)
  - [DAG](https://fivetran.github.io/dbt_aws_cloud_cost/#!/overview?g_v=1)
  - [Changelog](https://github.com/fivetran/dbt_aws_cloud_cost/blob/main/CHANGELOG.md)

## What does this dbt package do?
This package enables you to monitor and investigate cost & usage of different AWS services across your organizations. It creates enriched models with metrics focused on billing, pricing, line item buckets, products, reservations, and savings plans.

> > This package does not apply freshness tests.

Reviewer: ```suggestion
```

| [aws_cloud_cost__daily_instance_report](https://fivetran.github.io/dbt_aws_cloud_cost/#!/model/model.aws_cloud_cost.aws_cloud_cost__daily_instance_report) | Analyzes daily costs and usage for EC2 instances across all accounts to optimize compute spending and identify underutilized or oversized instances. <br></br>**Example Analytics Questions:**<ul><li>Which EC2 instance types generate the highest daily costs by account?</li><li>How does EC2 instance usage and spending vary across different accounts or environments?</li><li>Are there opportunities to right-size instances based on usage patterns?</li></ul>|

¹ Each Quickstart transformation job run materializes these models if all components of this data model are enabled. This count includes all staging, intermediate, and final models materialized as `view`, `table`, or `incremental`.

---

## Prerequisites
To use this dbt package, you must have the following:

> - At least one Fivetran Aws Cloud Cost connection syncing data into your destination.

Reviewer: ```suggestion
- At least one Fivetran AWS Cloud Cost connection syncing data into your destination.
```

- A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **PostgreSQL** destination.

## How do I use the dbt package?
You can either add this dbt package in the Fivetran dashboard or import it into your dbt project:

> - To add the package in the Fivetran dashboard, follow our [Quickstart guide](https://fivetran.com/docs/transformations/dbt#transformationsfordbtcore).

Reviewer: ```suggestion
- To add the package in the Fivetran dashboard, follow our [Quickstart guide](https://fivetran.com/docs/transformations/data-models/quickstart-management#quickstartmanagement).
```

- To add the package to your dbt project, follow the setup instructions in the dbt package's [README file](https://github.com/fivetran/dbt_aws_cloud_cost/blob/main/README.md#how-do-i-use-the-dbt-package) to use this package.

> > DISCLAIMER: This package transforms source data of potentially very high volumes. Please be aware of the size of your dataset(s) and take this into consideration when configuring the frequency with which you will orchestrate the package models. See [Additional configurations](https://github.com/fivetran/dbt_aws_cloud_cost?tab=readme-ov-file#optional-step-4-additional-configurations) for tools to mitigate compute and storage costs.

Reviewer: ```suggestion
> DISCLAIMER: This package transforms source data of potentially very high volumes. Please be aware of the size of your dataset(s) and take this into consideration when configuring the frequency with which you will orchestrate the package models. See [Additional configurations](https://github.com/fivetran/dbt_aws_cloud_cost?tab=readme-ov-file#optional-additional-configurations) for tools to mitigate compute and storage costs.
```
